_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
+
+- **Managed-service backups**: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for both {{ site.data.products.db }} clusters.
```

Comment:
Recommend slight tweaking:

For serverless and dedicated clusters, Cockroach Labs takes automated backups so that customers' don't have to manage backups themselves. These backups are stored on Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page to learn more about the type and frequency of backups supported on Dedicated and Serverless clusters.

Reply:
General question - Is the standard across the Docs to represent it as "severless" and "dedicated" (lowercase), as opposed to "Serverless" and "Dedicated" (upper case)?

Reply:
Marketing decided that these should be lowercased a month or two back, so docs needed to follow.